src/openpersonen/api/tests/test_data/ingeschreven_persoon.py

```diff
@@ -58,7 +58,7 @@
         "geboorte": {
             "_embedded": {
                 "datum": {"dag": 15, "datum": "19800915", "jaar": 1980, "maand": 9},
-                "land": {"code": "Nederland", "omschrijving": "Nederland"},
+                "land": {"code": "6030", "omschrijving": "Nederland"},
```

Comment:
Shouldn't this be an integer?
```suggestion
                "land": {"code": 6030, "omschrijving": "Nederland"},
```

Reply:
According to VNG the code should be returned as a string.  

A bit odd but they do give 
```
"land": {
   "code": "6030",
   "omschrijving": "Nederland"
}
```
as an example.